packages/cli/test/unit/commands/login.test.ts

```diff
@@ -47,19 +47,44 @@ describe('login', () => {
       await expect(exitCodePromise).resolves.toEqual(0);
     });
 
-    it('should allow the `--text-only` flag', async () => {
+    it('should allow the `--no-color` flag', async () => {
       const user = useUser();
-      client.setArgv('login', '--text-only');
+      client.setArgv('login', '--no-color');
       const exitCodePromise = login(client);
-      await expect(client.stderr).toOutput(`> Log in to Vercel`);
+      await expect(client.stderr).toOutput(`? Log in to Vercel`);
 
       // Move down to "Email" option
       client.stdin.write('\x1B[B'); // Down arrow
       client.stdin.write('\x1B[B'); // Down arrow
       client.stdin.write('\x1B[B'); // Down arrow
       client.stdin.write('\r'); // Return key
 
-      await expect(client.stderr).toOutput('> Enter your email address:');
+      await expect(client.stderr).toOutput('? Enter your email address:');
+
+      client.stdin.write(`${user.email}\n`);
+
+      await expect(client.stderr).toOutput(
+        `Success! Email authentication complete for ${user.email}`
+      );
+
+      await expect(client.stderr).not.toOutput(emoji('tip'));
+
+      await expect(exitCodePromise).resolves.toEqual(0);
+    });
+    it('should remove image the `NO_COLOR` env var with 1', async () => {
+      process.env.NO_COLOR = '1';
```

Comment:
**suggestion (blocking):** set and unset `NO_COLOR` in `before`/`after` hooks

To avoid test pollution, can we set `process.env.NO_COLOR = '1';` in a `before` hook and unset it in an `after` hook?